tests/general/LineLengthBearTest.py

```diff
@@ -26,3 +30,29 @@
     settings={
         'max_line_length': '4',
         'ignore_length_regex': 'http://, https://, ftp://'})
+
+
+LineLengthBearAspectTest = verify_local_bear(
+    LineLengthBear,
+    valid_files=(test_file,),
+    invalid_files=('testa',
+                   'test line'),
+    aspects=AspectList([
+        get_aspect('LineLength')('Unknown', max_line_length=4),
+        get_aspect('Indentation')('Unknown', indent_size=2),
```

Comment:
including this aspect is confusing.

The test files do not use tabs, so this is irrelevant, and the bear has a default value, so it isnt needed here.

Create a separate test for the indent aspect.

Reply:
Yes, you are right, seems like `LineLengthBear` is not for indentation. Even doing `settings={'indent_size': '4'}` gives an error. I am removing this aspect from this bear. :+1: